specification/consumption/resource-manager/Microsoft.Consumption/2017-11-30/consumption.json

```diff
@@ -156,7 +156,8 @@
             "in": "query",
             "required": true,
             "type": "string",
-            "enum": ["daily", "monthly"]
+            "enum": ["daily", "monthly"],
+            "modelAsString":true
```

Comment:
please take a look at https://github.com/Azure/autorest/blob/master/docs/extensions/readme.md#x-ms-enum
there's a specific format to follow to add the x-ms-enum extension.

Reply:
Done